lms/djangoapps/badges/events/course_complete.py

```diff
@@ -0,0 +1,63 @@
+"""
+Helper functions for the course complete event that was originally included with the Badging MVP.
+"""
+import hashlib
+
+from django.core.urlresolvers import reverse
+from django.template.defaultfilters import slugify
+from django.utils.translation import ugettext_lazy as _
+
+from badges.utils import site_prefix
+
+
+# NOTE: As these functions are carry-overs from the initial badging implementation, they are used in
+# migrations. Please check the badge migrations when changing any of these functions.
```

Comment:
I don't understand this comment: I couldn't find any use of any of these functions in migrations.  Are they used indirectly? Is this comment out of date?